src/bindings/js/node/include/infer_request.hpp

```diff
@@ -16,7 +16,7 @@ struct TsfnContext {
     Napi::Promise::Deferred deferred;
     Napi::ThreadSafeFunction tsfn;
 
-    ov::InferRequest* _ir;
+    ov::InferRequest* _ir = nullptr;
```

Comment:
Why is required?
Using default ctor it should be nullptr. Or is issue when custom ctor used `TsfnContext(Napi::Env env)`?
If used just add all members to initializer list to be sure that they will be initialized in expected way.

Reply:
This issue (136543) is the reason why I added obvious assignment, especially checks with error: 'Uninitialized pointer field'. Do you propose to move initialization to structure initializer?